docs/predictive-analytics/next-best-offer/process.sql

```diff
@@ -0,0 +1,82 @@
+-- Setup initial stream/table from Kafka topics
```

Comment:
This comment doesn't correspond to the command below.  Either move comment down or delete